config/config.php

```diff
@@ -82,6 +84,8 @@
 
     'enable.http_post' => false,
 
+    'signature.algorithm' => $saml2auth->config->signaturealgorithm,
```

Comment:
Should we check if 'signaturealgorithm' set and not empty here? and also get a default one if it's not set or empty for some reason?

Reply:
Sounds like a sensible thing to do. I will add that.